include/crow/json.h

```diff
@@ -1160,7 +1160,7 @@ namespace crow
         ///
         /// Value can mean any json value, including a JSON object.
         /// Write means this class is used to primarily assemble JSON objects using keys and values and export those into a string.
-        class wvalue
+        class wvalue //TODO attempting to make this returnable resulted in a linker error, any help is appreciated
```

Comment:
I've moved `returnable` type to separate file, included it in `json.h` and `multipart.h` and it looks OK (tests build with no error).

Reply:
Interesting, I'll look into it